doc-content/enterprise-only/cases/case-management-milestone-triggering-proc.adoc

```diff
@@ -9,24 +9,24 @@ Case milestones can be configured to start automatically when a case instance st
 * A case definition has been created.
 
 .Procedure
-. In the process designer, click image:cases/3898.png[right arrows] to open the *Object Library* on the left of the design palette.
-. From the list, open *Milestone* and drag the `Milestone` object onto the palette.
+. From the predefined node panel located on the left side of the diagram editor, drag and drop a `Milestone` object onto the palette.
 +
 image::cases/milestone.png[Milestone]
 +
-. With the milestone node selected, click image:cases/3897.png[left arrows] to open the *Properties* panel on the right.
-. Open *Assignments* to add a completion condition. Milestones include a `condition` parameter by default.
-. To define the completion condition to for the milestone, select `Constant` from the list. The condition must be provided using the Drools syntax.
-. Configure the `Adhoc autostart` property.
+
+. With the milestone selected, click image:getting-started/diagram_properties.png[] to open the *Properties* panel on the right side of the designer.
+. Expand *Data Assignments* to add a completion condition. Milestones include a `condition` parameter by default.
+. To define the completion condition to for the milestone, select `Constant` from the `Source` list. The condition must be provided using the Drools syntax.
```

Comment:
Probably omit `to`?
To define the completion condition *to for* the milestone

Reply:
Corrected. Thank you:)